plans/main.fmf

```diff
@@ -1,40 +1,36 @@
-## Note: "order" is set to 50 by default. Orders with lower values are
-## prioritized.
-## Ref: https://tmt.readthedocs.io/en/stable/spec/core.html#order
+prepare+:
+    - name: Install bats
+      how: shell
+      script: |
+        BATS_VERSION=1.11.0
+        curl -s -L -O https://github.com/bats-core/bats-core/archive/refs/tags/v$BATS_VERSION.tar.gz
+        tar zxf v$BATS_VERSION.tar.gz
+        cd bats-core-$BATS_VERSION
+        ./install.sh /usr
```

Comment:
Is there a strong reason to do this? I rather use the packaged version. Is the issue that there is no epel 10 yet?

Reply:
I'm not sure what you mean by "`parallel` support". As best I understand, plain BATS will use GNU parallel just fine. What I see in your link above is that the bats rpm has a `Requires: parallel` -- that's something we could do ourselves.

For now and the immediate future this is moot, since system tests can't yet run in parallel.

Reply:
> I'm not sure what you mean by "`parallel` support". As best I understand, plain BATS will use GNU parallel just fine. What I see in your link above is that the bats rpm has a `Requires: parallel` -- that's something we could do ourselves.
> 

@edsantiago ack, I just saw `BuildRequires: parallel` in there so I thought `support`.

> For now and the immediate future this is moot, since system tests can't yet run in parallel.

In that case I would prefer what I currently have on the PR, mainly for simplicity.

Reply:
FYI we are in aardvark-dns here, we do not have parallel runs enabled but the tests are designed with sufficient isolation that it could be easily be enabled I think. 
Test timing is not that critical on this repo but I think losing this ability is bad.

Reasons why I don't like this:
- nobody ever will remember to update this there as such we keep using outdated bats forever and then don't notice if things break
- there is no trust chain, with the rpm package we get gpg signatures, sure this likely does not matter as upstream must be trusted no matter what
- there is no way to know what we get really with executing the install.sh script? What if it fails for some reason? We know have to debug this thing with no knowledge what it executes.

Reply:
FWIW, this (fetch & unpack static tarball from github) is the method used in RHEL gating tests. It is unfortunately more reliable than epel or anything else I could think of, particularly on unreleased RHELs.

Reply:
1 possible alternative I could try would be `dnf install $BATS_RPM_URL $PARALLEL_RPM_URL` where the URLs are generic enough that they'll work for all distros without any extra setup. I'll have to pull in parallel too because of the install requirement. I can check on that